src/layers/core/hexagon-layer/hexagon-layer.js

```diff
@@ -125,6 +121,18 @@ export default class HexagonLayer extends Layer {
     }
   }
 
+  getHexagons() {
+    const {hexagonAggregator} = this.props;
+    const {viewport} = this.context;
+    const hexagons = hexagonAggregator(this.props, viewport);
+    Object.assign(this.state, {hexagons});
```

Comment:
Nit: we have a `Layer.setState` method that also sets the dirty flag to ensure re-render.